### ISSUE TITLE
Reader: Tapping a YouTube video title should open the YouTube app

### ISSUE THREAD

**Issue by twstokes** [wordpress-mobile/WordPress-iOS#19598]
### Expected behavior
The YouTube app opens with the video.

### Actual behavior
Nothing happens.

### Steps to reproduce the behavior
1. Ensure the YouTube app is installed.
2. Embed a YouTube video in a Post.
3. Load the Post in Reader in the WordPress or Jetpack app.
4. Tap the title of the YouTube video.

<img src="https://user-images.githubusercontent.com/2092798/201980457-4174ac3c-44c7-4f0a-9b6b-fbd3ae742481.png" width="300px" />

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by jkmassel** [comment]
I don't think this is our bug? I'd expect the YouTube app to need to handle YouTube URLs? 

I'm going to close this one until/unless we get some substantial user feedback about it – if so we can dig into it (because I don't think it's trivial).
